platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetTracker.java

```diff
@@ -78,6 +81,12 @@ public TipsetTracker(@NonNull final Time time, @NonNull final AddressBook addres
         tipsets = new StandardSequenceMap<>(0, INITIAL_TIPSET_MAP_CAPACITY, true, EventDescriptor::getGeneration);
 
         ancientEventLogger = new RateLimitedLogger(logger, time, Duration.ofMinutes(1));
+
+        if (ancientMode == AncientMode.BIRTH_ROUND_THRESHOLD) {
+            this.nonAncientEventWindow = NonAncientEventWindow.INITIAL_EVENT_WINDOW_BIRTH_ROUND;
+        } else {
+            this.nonAncientEventWindow = NonAncientEventWindow.INITIAL_EVENT_WINDOW_GENERATION;
+        }
```

Comment:
Use helper function for getting initial window here (if you add it)

Reply:
Suggestion adopted, code is much cleaner now.